signup-service/src/main/java/io/mosip/signup/services/IdentityVerificationService.java

```diff
@@ -178,4 +206,19 @@ private IdentityVerifierDetail[] getIdentityVerifierDetailsFromConfigServer() {
         }
         return verifierDetails;
     }
+
+    private PrivateKey loadPrivateKey(String alias, String cyptoPassword) {
+        try {
```

Comment:
Lets use try-with-resource so that inputstream is automatically closed.

Reply:
done